qiita_pet/test/test_portal.py

```diff
@@ -0,0 +1,58 @@
+from unittest import main
+from qiita_pet.test.tornado_test_base import TestHandlerBase
+
+from mock import Mock
+
+from qiita_db.user import User
+from qiita_pet.handlers.base_handlers import BaseHandler
+
+
+class TestPortal(TestHandlerBase):
+    def test_get(self):
+        BaseHandler.get_current_user = Mock(return_value=User("admin@foo.bar"))
+        response = self.get('/admin/portals/studies/')
+        self.assertEqual(response.code, 200)
```

Comment:
"Funny" enough there is nothing dynamic created in this page as it's retrieved via AJAX, and JS is not executed. Everything is created in /admin/portals/studiesAJAX/, for example: /admin/portals/studiesAJAX/?sEcho=1001&view-portal=QIITA ...